client/search-ui/src/components/index.ts

```diff
@@ -1,3 +1,13 @@
-export * from './SyntaxHighlightedSearchQuery'
+export * from './CodeHostIcon'
 export * from './CommitSearchResult'
+export * from './CommitSearchResultMatch'
+export * from './FileSearchResult'
+export * from './LastSyncedIcon'
+export * from './RepoFileLink'
 export * from './RepoSearchResult'
+export * from './ResultContainer'
+export * from './SearchResultStar'
+export * from './SyntaxHighlightedSearchQuery'
+
```

Comment:
```suggestion
import styles from './FileMatchChildren.module.scss'
export { styles as FileMatchChildrenStyles }
```

The VS Code extension is using the FileMatchChildren.module.scss [here](https://github.com/sourcegraph/sourcegraph/blob/1c7eb1f044d6acb5b54a9e759530224d03aa7fa4/client/vscode/src/webview/search-panel/alias/FileMatchChildren.tsx#L19), is it possible to export it as well?